src/main/macro/Importer.ijm

```diff
@@ -0,0 +1,42 @@
+/*
+ * #%L
+ * OME Bio-Formats package for reading and converting biological file formats.
+ * %%
+ * Copyright (C) 2019 Open Microscopy Environment:
+ *   - Board of Regents of the University of Wisconsin-Madison
+ *   - Glencoe Software, Inc.
+ *   - University of Dundee
+ * %%
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
Is the usage of the BSD license done purposefully? Currently, this component is depending on `formats-gpl` and under the terms of the GPL license. That being said, I do not think anything in the current state of this repository requires the GPL code so depending on `formats-bsd` only and relicensing might be open for consideration.

Reply:
copy paste from other example.